README.md

```diff
@@ -1,105 +1,168 @@
-Censys Python Library
-=====================
+# Censys Python Library ![PyPI](https://img.shields.io/pypi/v/censys) ![Python Versions](https://img.shields.io/pypi/pyversions/censys)
 
-This is a light weight Python wrapper to the Censys REST API.
+This is a light weight Python wrapper to the Censys REST API. `censys-python` currently supports Python 3.6+.
 
-Install
--------
+## Install
 
-The egg can be installed using Pip or easy_install (e.g., `sudo pip install censys`).
+The egg can be installed using `pip` or `easy_install`
 
-Usage
-=====
+```bash
+pip install censys
+```
+
+## Usage
 
 There are three API endpoints that the library provides access to: Index,
```

Comment:
@bwireman Here we reference Index, Query, and Export. Do we still use these terms?